main/templates/base/form_buttons.html

```diff
@@ -3,37 +3,43 @@
         {% if not no_back %}
-            <input class="button" type="submit" name="save_back" value="{% trans '<< Opslaan en vorige stap' %}"/>
+            <input class="button"
+                   type="submit"
+                   name="save_back"
+                   value="{% trans '<< Opslaan en vorige stap' %}" />
             {% if proposal %}
                 {% if proposal.is_pre_assessment %}
-                    <a class="button js-submit-redirect" href="{% url 'proposals:update_pre' proposal.pk %}">
-                        {% trans "Terug naar begin aanvraag" %}
-                    </a>
+                    <a class="button js-submit-redirect"
+                       href="{% url 'proposals:update_pre' proposal.pk %}">{% trans "Terug naar begin aanvraag" %}</a>
                 {% elif proposal.is_practice %}
-                    <a class="button js-submit-redirect" href="{% url 'proposals:update_practice' proposal.pk %}">
+                    <a class="button js-submit-redirect"
+                       href="{% url 'proposals:update_practice' proposal.pk %}">
                         {% trans "Terug naar begin aanvraag" %}
                     </a>
                 {% else %}
-                    <a class="button js-submit-redirect" href="{% url 'proposals:update' proposal.pk %}">
-                        {% trans "Terug naar begin aanvraag" %}
-                    </a>
+                    <a class="button js-submit-redirect"
+                       href="{% url 'proposals:update' proposal.pk %}">{% trans "Terug naar begin aanvraag" %}</a>
                 {% endif %}
             {% endif %}
             {% if study %}
-                <a class="button js-submit-redirect" href="{% url 'studies:update' study.pk %}">
-                    {% trans "Terug naar begin traject" %}
-                </a>
+                <a class="button js-submit-redirect"
+                   href="{% url 'studies:update' study.pk %}">{% trans "Terug naar begin traject" %}</a>
             {% endif %}
             {% if session %}
-                <a class="button js-submit-redirect" href="{% url 'tasks:start' session.pk %}">
-                    {% trans "Terug naar begin sessie" %}
-                </a>
+                <a class="button js-submit-redirect"
+                   href="{% url 'tasks:start' session.pk %}">{% trans "Terug naar begin sessie" %}</a>
             {% endif %}
         {% endif %}
         {% if not no_forward %}
-            <input class="button button-colored continue-button" type="submit" name="save_continue"
-                   value="{{ next_text|default:_('Opslaan en volgende stap >>') }}"/>
+            <input class="button button-colored continue-button"
+                   type="submit"
+                   name="save_continue"
+                   value="{{ next_text|default:_("
+                   Opslaan
+                   en
+                   volgende
+                   stap>
+            >") }}" />
```

Comment:
Somethings's wrong here

Reply:
Interesting... Looks like DJLint has a bug when it comes to the `_('[..]')` syntax. 

Should be fixed now, I moved the translated string to a variable and referenced that in the default filter